src/screens/About/About.tsx

```diff
@@ -21,27 +21,31 @@ export const sidebarItems: TocItem[] = [
     items: [
       {
         to: '/about#mission',
-        label: 'Our mission',
```

Comment:
I can undo these changes if we're not a fan, but all the text matched exactly so I figured I might as well remove the redundancy.